setup/src/Magento/Setup/Controller/Navigation.php

```diff
@@ -79,11 +81,12 @@ public function menuAction()
 
     /**
      * @return array|ViewModel
+     * @throws Exception
      */
     public function sideMenuAction()
     {
-        /** @var \Magento\Backend\Model\UrlInterface $backendUrl */
-        $backendUrl = $this->objectManagerProvider->get()->get(\Magento\Backend\Model\UrlInterface::class);
+        /** @var UrlInterface $backendUrl */
+        $backendUrl = $this->objectManagerProvider->get()->get(UrlInterface::class);
```

Comment:
You should inject that class to the constructor.

Reply:
Hi @lbajsarowicz ,
Unfortunately I can't inject `\Magento\Backend\Model\UrlInterface`directly to constructor. I get such error.
<img width="1433" alt="Screenshot 2020-02-03 at 18 24 27" src="https://user-images.githubusercontent.com/13517181/73672486-9e795c80-46b5-11ea-90b4-088597229a22.png">

Also I want to notice that construction  like this  `$this->objectManagerProvider->get()` is used in a lot of cases. 

If such approach is wrong, maybe You could advise me something ?

Thanks a lot!